.github/workflows/release_macOS.yml

```diff
@@ -51,6 +51,15 @@ jobs:
         run: yarn release:mac:x64
         env:
           NOTARIZE: true
+      - name: Copy dist/latest-mac.yml to dist/latest-mac-x64.yml
+        run: |
+          cp dist/latest-mac.yml dist/latest-mac-x64.yml
+        shell: bash
+      - name: Save latest-mac-x64.yml as an artifact
+        uses: actions/upload-artifact@v2
```

Comment:
there is a v4 version of this action https://github.com/actions/upload-artifact?tab=readme-ov-file#v4---whats-new